README.md

```diff
@@ -44,11 +44,38 @@ Then edit the `.env` file to set your specific configuration values.
 
 ### Starting the services
 
+You can start the services in two ways, depending on your environment:
+
+#### 1. Development (default)
+
+By default, Docker Compose uses `docker-compose.yml` (and `docker-compose.override.yml` if present):
+
+```bash
+docker compose up -d
+```
+
+#### 2. Production (with production overrides)
+
+For production, you can use a separate override file (e.g., `docker-compose.prod.yml`). There are two ways to do this:
```

Comment:
@anatolyshipitz let's avoid excessive documentation. Could you please provide exact mechanism how this this stuff is deployed. e.g. I suppose it uses -f option.